### ISSUE TITLE
ParenthesesAroundCondition bug

### ISSUE THREAD

**Issue by bbatsov** [rubocop/rubocop#127]
The following code(from `TrivialAccessors`) does not generate an offence:

``` ruby
if (is_trivial_reader(sexp, accessor_var) ||
    is_trivial_writer(sexp, accessor_var))
  add_offence(:convention, lineno, ERROR_MESSAGE)
end
```

@jonas054, maybe you can see what the problem is, since you wrote that cop?
